tests/test_apm.py

```diff
@@ -27,8 +27,12 @@
 from .compat import StringIO
 
 
-def test_elasticapm_structlog_log_correlation_ecs_fields(spec_validator):
-    apm = elasticapm.Client({"SERVICE_NAME": "apm-service", "DISABLE_SEND": True})
+# elasticapm only supports python 3.6+
+if sys.version_info < (3, 6):
```

Comment:
Can we move this skip within the `apm` fixture possibly?

Reply:
I hadn't realized you could do a skip inside of a fixture. I'll do that for sure.